tool_commands/raster_checker/raster_checker_main.py

```diff
@@ -14,11 +14,19 @@
 from qgis.PyQt.QtCore import QVariant
 from sqlalchemy import MetaData
 from sqlalchemy.ext.declarative import declarative_base
-from ThreeDiToolbox.utils.constants import RASTER_CHECKER_MAPPER
-from ThreeDiToolbox.utils.raster_checker_log import RasterCheckerProgressBar
-from ThreeDiToolbox.utils.raster_checker_log import RasterCheckerResults
-from ThreeDiToolbox.utils.raster_checker_prework import DataModelSource
-from ThreeDiToolbox.utils.raster_checker_prework import RasterCheckerEntrees
+from ThreeDiToolbox.tool_commands.raster_checker.constants import RASTER_CHECKER_MAPPER
+from ThreeDiToolbox.tool_commands.raster_checker.raster_checker_log import (
```

Comment:
Dit zijn hele lange imports.

In hoeverre is `from ThreeDiToolbox.tool_commands.raster_checker import raster_checker_log` mogelijk? Dat je dan verderop `raster_checker_log.RasterCheckerProgressBar` gebruikt? Dan verspreiden we de pijn van de lengte een beetje.

Misschien is het ook duidelijker waar iets vandaan komt? Is dat een idee?

Reply:
goede tip!